plugins/pagerduty/src/api/client.ts

```diff
@@ -124,11 +130,13 @@ export class PagerDutyClient implements PagerDutyApi {
   }
 
   private async getByUrl<T>(url: string): Promise<T> {
+    const { token: idToken } = await this.config.identityApi.getCredentials();
     const options = {
       method: 'GET',
       headers: {
         Accept: 'application/vnd.pagerduty+json;version=2',
         'Content-Type': 'application/json',
+        ...(idToken && { Authorization: `Bearer ${idToken}` }),
```

Comment:
I think that this is going to use the `backstageToken` for requests that's possibly go to `'https://events.pagerduty.com/v2'` if there is no `pagerDuty.eventsBaseUrl`. Do you have your set to a `/proxy` endpoint by any chance?

Reply:
`eventsBaseUrl` is only used in the `triggerAlarm` function, whereas the functions calling `getByUrl` are calling the proxy so it should be fine, unless I'm misunderstanding your question

Reply:
Ah ok - sorry I misread the code here. Ok, so this is because you have the middleware to validate the token on top of the proxy backend plugin right?